make/RunTests.gmk

```diff
@@ -539,12 +539,11 @@ define SetupRunGtestTestBody
 	  $$(if $$($1_PASSED), , $$(eval $1_PASSED := 0)) \
 	  $$(eval $1_SKIPPED := $$(shell $$(AWK) \
 	      '/YOU HAVE [0-9]+ DISABLED TEST/ { \
-	          if (match($$$$0, /[0-9]+/, arr)) { \
-	            print arr[0]; \
+	          if (match($$$$0, /YOU HAVE ([0-9]+) DISABLED TEST/, arr)) { \
+	            print arr[1]; \
```

Comment:
I can't see how this change is needed. The block after the `/.../` regex will only be executed for lines matching the regex.

Reply:
You are right, these two line change has been reverted. Thanks.